### PR TITLE
Download vcard on click

### PR DESCRIPTION
**Pull request by foucdeg**
Click line to download VCard

### CODE REVIEW

File: src/components/PeopleList.js [hunk 1]
```diff
@@ -4,19 +4,22 @@ import { ListItem, Link } from 'react-toolbox';
...
     const legend = computeLegend(someone, this.props.companyEmails);
+    const onItemClick = this.exportVcard.bind(this, someone, this.props.companyEmails);
     return (
       <ListItem
         key={someone.name}
         avatar={someone.avatar}
         caption={someone.name}
         legend={legend}
         rightIcon={phoneCallToAction}
+        onClick={onItemClick}
```
Comment (tcheymol) [inline src/components/PeopleList.js]: onClick={() => exportVcard(someone, this.props.companyEmails)}

Comment (tcheymol) [reply src/components/PeopleList.js]: So that you can remove the above binding

File: src/services/People.js [hunk 2]
```diff
@@ -77,21 +82,43 @@ const searchPeople = (searchText) => {
...
     }
-    return matchingCompanies;
+    return _matchingCompanies;
   }, []);
   return matchingCompanies;
 };
 
+const buildCompanyName = (companyEmail) => {
+  if (companyEmail === '@theodo.co.uk') return 'Theodo UK';
+  if (companyEmail === '@bam.tech') return 'BAM';
```
Comment (tcheymol) [inline src/services/People.js:101]: Il ne manque pas 3 boites ?

**Review by tcheymol — DISMISSED** [review]

File: src/components/PeopleList.js (hunk 1 above)
Comment (foucdeg) [reply src/components/PeopleList.js]: OK, won't help performance though

File: src/services/People.js (hunk 2 above)
Comment (foucdeg) [reply src/services/People.js:101]: cf ligne du dessous :)

**Pull request merged**